google/self_link_helpers.go

```diff
@@ -28,6 +28,17 @@ func compareSelfLinkRelativePaths(k, old, new string, d *schema.ResourceData) bo
 	return false
 }
 
+// This method should only be used if the self link references a global resource.
+func compareSelfLinkResourceNames(k, old, new string, d *schema.ResourceData) bool {
```

Comment:
Wait I'm confused why `compareSelfLinkRelativePaths` doesn't work on global resources.

Network self links look like https://www.googleapis.com/compute/v1/projects/project/global/networks/network, so comparing the relative one would be comparing `project/global/networks/network` which should work just fine, right?